src/test/java/com/monumental/services/integrationtest/MonumentServiceMockIntegrationTests.java

```diff
@@ -186,7 +186,7 @@ public void testMonumentService_BulkCreateFlow_OneInvalidCsvRecord() {
 
         CsvMonumentConverterResult validationErrors = validationResult.getInvalidResults().get(1);
 
-        assertEquals(3, validationErrors.getWarnings().size());
+        assertEquals(1, validationErrors.getWarnings().size());
         assertTrue(validationErrors.getWarnings().contains("Please use decimal coordinates, not degrees. To convert, input your degrees into Google Maps."));
```

Comment:
We could import the string from `CsvMonumentConverter` here and use that for `.contains()`